orga/app/controllers/authenticated/campaigns/campaign/analysis.js

```diff
@@ -0,0 +1,10 @@
+import Controller from '@ember/controller';
+import { service } from '@ember/service';
+
+export default class AnalysisController extends Controller {
+  @service currentUser;
+
+  get isGARAuthenticationMethod() {
```

Comment:
Pour suivre les conventions de nommage : 
```suggestion
  get isGarAuthenticationMethod() {
```

Reply:
On pourrait profiter de cette PR pour faire le renommage `currentUser.isGARAuthenticationMethod`→`currentUser.isGarAuthenticationMethod`